src/barril/units/posc.py

```diff
@@ -321,6 +321,7 @@ def FillUnitDatabaseWithPosc(db=None, fill_categories=True, override_categories=
         "moment per angular velocity", "newton meter per angular velocity", "Nms/rad"
     )
     db.AddUnitBase("mass temperature per mol", "kg.K/mol", "kg.K/mol")
+    db.AddUnitBase("temperature per pressure", "K/Pa", "K/Pa")
     f_unit_to_base = MakeCustomaryToBase(0.0, 6.283185307, 1.0, 0.0)
```

Comment:
@tadeu please look at this one. The (conversion) functions have the same arguments... this is the reason why I did what I did.

Reply:
Ok, no problem